client/src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Nicee 👌 

Are there any additional files that need to be committed to get the project running with `bootstrap` or a local setup required to use that distro of the package?

We could alternatively use `yarn add bootstrap` to add the package to the project if it's on the same distro as the one referenced here.

Reply:
You are right, updated the dependencies in the package and yarn lock files